stdlib/distutils/command/install.pyi

```diff
@@ -4,8 +4,8 @@ from typing import Any, ClassVar, Final
 
 from ..cmd import Command
 
-HAS_USER_SITE: bool
-SCHEME_KEYS: tuple[str, ...]
+HAS_USER_SITE: Final[bool]
+SCHEME_KEYS: Final = ("purelib", "platlib", "headers", "scripts", "data")
```

Comment:
Well looks like pytype doesn't like this syntax (and I don't see an issue open for it https://github.com/google/pytype/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen+final+tuple )
```suggestion
SCHEME_KEYS: Final[tuple["purelib", "platlib", "headers", "scripts", "data"]]
```